src/values/ObjectValue.js

```diff
@@ -325,6 +325,11 @@ export default class ObjectValue extends ConcreteValue {
   // never be serialized.
   refuseSerialization: boolean;
 
+  // Checks whether effects are properly applied.
```

Comment:
I don't see anything that ensures that _isPartial is only true when effects are applied/undone. I thought this was an annotation used in our models/used for snapshotting.